infra/resources/prod/key_vault.tf

```
@@ -0,0 +1,17 @@
+module "key_valuts" {
+  source = "../_modules/key_vaults"
+
+  resource_group_name = azurerm_resource_group.itn_com.name
+
+  tags = local.tags
+
+  environment = {
+    prefix      = "io"
+    location    = local.location
+    name        = "com",
+    domain      = "",
+    environment = "p",
+  }
```

These values must not be hardcoded, use the appropriate `local` here.

And there is no need to pass a blank "domain" if we don't want to value it, just remove this field from `key_vaults/variables.tf`

```suggestion
  environment = {
    prefix      = local.prefix
    location    = local.location
    name        = local.domain,
    environment = local.env_short,
  }
```